src/schema/notifications.ts

```diff
@@ -126,6 +128,10 @@ export const typeDefs = /* GraphQL */ `
   }
 
   extend type Query {
+    """
+    Get the active notification count for a user
+    """
+    notificationCount: Int @auth
```

Comment:
I think it's a better name

```suggestion
    unreadNotificationsCount: Int @auth
```